api/tests/integration/scripts/prod/archive-campaigns_test.js

```diff
@@ -16,7 +16,15 @@ describe('Script | Prod | Archive Campaign', function () {
   describe('#archiveCampaigns', function () {
     it('archives all given campaigns', async function () {
       const file = './tests/tooling/fixtures/campaigns-to-archive.csv';
-      const logger = { setTotal: () => {}, log: () => {}, logCount: () => {} };
+
+      const logger = {
+        // eslint-disable-next-line no-empty-function
+        setTotal: () => {},
+        // eslint-disable-next-line no-empty-function
+        log: () => {},
+        // eslint-disable-next-line no-empty-function
+        logCount: () => {},
+      };
```

Comment:
On peut expliciter et eviter le linter:
```suggestion
      const logger = {
        // eslint-disable-next-line no-empty-function
        setTotal: _.noop,
        // eslint-disable-next-line no-empty-function
        log: () => _.noop,
        // eslint-disable-next-line no-empty-function
        logCount: _.noop,
      };
```

Reply:
Corrigé